src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerFileReferenceTests.cs

```diff
@@ -250,7 +250,8 @@ public void TestFailedLoadDoesntCauseNoAnalyzersWarning()
             Assert.Equal(AnalyzerLoadFailureEventArgs.FailureErrorCode.UnableToCreateAnalyzer, errors.First().ErrorCode);
         }
 
-        [Fact]
+        // can't load a framework targeting generator, which these are in desktop
```

Comment:
>, which these are in desktop [](start = 53, length = 28)

I didn't really understand this part of the comment. Can it be removed? It seems like the previous statement is sufficient.

Reply:
Ah, it just means that when running the desktop version of the test, the generators we reference are also desktop (because they're in the same assembly). I'll clean it up to make it more clear.